docs/architecture/porting-existing-aspnet-apps/migration-considerations.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migration considerations
+description: What does a team need to know in order to make the right decision about whether and how to migrate from ASP.NET MVC to .NET Core?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Migration considerations
```

Comment:
I'm leading with "The most fundamental question teams must answer when it comes to porting their apps to .NET Core is, should they port at all? In some cases, the best path forward may be to remain on the .NET Framework using ASP.NET MVC and/or Web API. In this chapter, we'll consider reasons why moving to .NET Core may make sense, and consider some scenarios and counterpoints for staying on .NET."

Are you thinking the order should be reversed and we should talk about why .NET is a great place be, and to stay, and after that talk about why they might want to move to Core? If they're reading the book, they're probably already thinking they need to move to Core, so that's why I lead with that. I probably should reference our Modernizing book here, too: https://docs.microsoft.com/en-us/dotnet/architecture/modernize-with-azure-containers/